requirements.txt

```diff
@@ -20,7 +20,7 @@ overrides
 nltk
 
 # Mainly used for the faster tokenizer.
-spacy
+spacy==2.0
```

Comment:
Any opposition to something like `spacy>=2.0,<2.1`?  Then we'll get bugfixes, but won't upgrade to a new minor version automatically, and have some flexibility with other libraries that pin their versions.  As spacy gets past 2.1, too, we can see if it breaks anything, and expand the bounds accordingly.